### DISCUSSION TITLE
Cant use ngx pdf viewer

### DISCUSSION THREAD

**Discussion by anoop666** [stephanrauh/ngx-extended-pdf-viewer#1465] [Q&A] [answered]
Failed to load resource the server responded with a status of 404  6200/assests/pdf-2.10.549.min.js.1

**Comment by anoop666** [comment]
![IMG_20220727_095229](https://user-images.githubusercontent.com/47858623/181161015-8e5327f8-1306-463a-8d3c-b30e20845a5a.jpg)

**Comment by stephanrauh** [comment, marked as answer]
I can only guess - but your screenshot indicates the JavaScript files are missing. Did you update the angular.json according to the documentation?

I recommend to hunt down the bug like so:
- Create a fresh new Angular project (`ng new my-favorite-project`)
- `cd my-favorite-project`
- `ng add ngx-extended-pdf-viewer`
- add the new component to the `app.component.html` (I've forgotten the name of the component, but you'll find it quickly - it's the new component added by the `ng add` command)

Now you've got a running Angular application featuring the PDF viewer. Compare your `angular.json` to the `angular.json` of the new reference project. If you don't spot the difference, compare the two projects with each other. 

Hope that helps!
Stephan

**Reply by anoop666** [reply]
There were some files missing in my angular.json. It just worked fine after fixing those . Thank you ❤️
